inspect/src/lib.rs

```diff
@@ -127,6 +138,60 @@ pub fn run(options: Options) -> anyhow::Result<()> {
             "{}",
             serde_json::to_string_pretty(&payload)?.to_colored_json_auto()?
         );
+    } else if options.history {
+        let branch = Reference::try_from(&urn)?;
+        match storage.reference(&branch) {
+            Ok(Some(reference)) => {
+                let mut tip = reference.peel_to_commit()?;
+
+                loop {
+                    let tree = tip.tree()?;
+                    let entry = tree.get(0).unwrap().id();
+                    let blob = storage
+                        .find_object(Box::new(entry))?
+                        .unwrap()
+                        .into_blob()
+                        .unwrap();
```

Comment:
This one would fail if the object is not a blob, which is unlikely, but possible.